cmd/tnijto/main.go

```diff
@@ -1,3 +1,66 @@
 package main
 
-func main() {}
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	utils "github.com/turbaszek/tnijto/pkg"
+	"log"
+	"net/http"
+	"time"
+)
+
+var config = utils.NewConfig()
+
+func main() {
```

Comment:
I'd add some kind of log information to indicate where to look for tnijto when it starts. It took me a while to find port and I didn't notice that app is up 😉

Reply:
I will add. Btw the information is in README.md but... it's not merged yet so not easily visible.